content/documentation/overview/index.adoc

```diff
@@ -131,11 +131,17 @@ This section lists all the validations that Kiali performs on all Istio configur
 |https://github.com/kiali/kiali/tree/master/business/checkers/destination_rules/multi_match_checker.go[source code, window="_blank"]
 |link:/files/validation_examples/001.yaml[001.yaml, window="_blank"]
 
-|Host doesn't have a valid service
+|This host has no matching workloads
 |error
 |When one destination rule has a host that doesn't exist. This checks against any workload, service names as well as service entries
-|https://github.com/kiali/kiali/tree/master/business/checkers/destination_rules/no_host_checker.go[source code, window="_blank"]
+|https://github.com/kiali/kiali/blob/master/business/checkers/destinationrules/no_dest_checker.go#L28[source code, window="_blank"]
 |link:/files/validation_examples/002.yaml[002.yaml, window="_blank"]
+
+|This subset's labels are not found from any matching host
```

Comment:
"not found from any" sounds a bit strange. maybe "on any ..." or "in any..." ?

Reply:
Changed in kiali/kiali too.